src/views/language/LanguageVitalityComputation.tsx

```diff
@@ -0,0 +1,155 @@
+import { LanguageData } from '../../types/LanguageTypes';
+
+/**
+ * Maps Ethnologue 2013 vitality levels to 0-9 scale
+ * Ethnologue 2013 scale: 1=National, 2=Regional, 3=Trade, 4=Educational, 5=Written, 6=Threatened, 7=Shifting, 8=Moribund, 9=Nearly Extinct, 10=Extinct
+ * Our scale: 9=National, 8=Regional, 7=Trade, 6=Educational, 5=Written, 4=Threatened, 3=Shifting, 2=Moribund, 1=Nearly Extinct, 0=Extinct
+ */
+export function getEthnologue2013Score(vitality: string): number | null {
+  if (!vitality) return null;
+
+  switch (vitality.toLowerCase()) {
+    case 'national':
+      return 9;
+    case 'regional':
+      return 8;
+    case 'trade':
+      return 7;
+    case 'educational':
+      return 6;
+    case 'written':
+      return 5;
+    case 'threatened':
+      return 4;
+    case 'shifting':
+      return 3;
+    case 'moribund':
+      return 2;
+    case 'nearly extinct':
+      return 1;
+    case 'extinct':
+      return 0;
```

Comment:
Looks like you are missing some values. Check the spreadsheet lookup results. You can add extra cases to be defensive.

<img width="639" height="570" alt="Image" src="https://github.com/user-attachments/assets/d9965697-62ae-4bfe-9533-73ba519a8ced" />
```suggestion
    case 'national':
      return 9;
    case 'provincial':
    case 'regional':
      return 8;
    case 'trade':
    case 'wider communication':
      return 7;
    case 'educational':
      return 6;
    case 'written':
    case 'developing':
      return 5;
    case 'vigorous':
    case 'threatened':
      return 4;
    case 'shifting':
      return 3;
    case 'moribund':
    case 'nearly extinct':
      return 2;
    case 'dormant':
      return 1;
    case 'extinct':
      return 0;
```